drivers/hmis/app/models/hmis/form/form_processor.rb

```diff
@@ -68,6 +69,24 @@ def run!(owner:, user:)
     owner.enrollment = enrollment_factory if owner.is_a?(Hmis::Hud::CustomAssessment)
   end
 
+  def related_records
```

Comment:
Doesn't the existing `def related_records` method return the same result? This is more straightforward, but they're doing the same thing, I think.

Reply:
Oh jeez, I missed that already existed. Thanks! Maybe we are missing the DuplicateMethods cop, I'll look into that